File: Beelina.APP/src/app/_models/errors/product-warehouse-stock-receipt-entry-not-exists.error.ts
import { IBaseError } from 'src/app/_interfaces/errors/ibase.error';
import { IProductWarehouseStockReceiptEntryPayload } from 'src/app/_interfaces/payloads/iproduct-warehouse-stock-receipt-entry-query.payload';

export class ProductWarehouseStockReceiptEntryNotExistsError
  implements IProductWarehouseStockReceiptEntryPayload, IBaseError {
  public typename: string;
  public code: string;
  public message: string;
}

File: Beelina.LIB/GraphQL/Errors/ProductWarehouseStockReceiptEntryNotExistsError.cs
using Beelina.LIB.Interfaces;

namespace Beelina.LIB.GraphQL.Errors
{
    public class ProductWarehouseStockReceiptEntryNotExistsError
        : BaseError, IProductWarehouseStockReceiptEntryPayload
    {
        public ProductWarehouseStockReceiptEntryNotExistsError(int id)
        {
            Message = $"Product warehouse stock receipt entry with the id of {id} does not exists!";
        }
    }
}
